src/marks/text.js

```diff
@@ -137,7 +137,9 @@ function applyMultilineText(selection, mark, T, TL) {
   if (!T) return;
   const {lineAnchor, lineHeight, textOverflow, splitLines, clipLine} = mark;
   selection.each(function (i) {
-    const lines = splitLines(formatDefault(T[i])).map(clipLine);
+    const t = formatDefault(T[i]);
+    if (t == null) return;
+    const lines = splitLines(t).map(clipLine);
```

Comment:
How about this, for brevity? I don’t think we need to optimize for the case where formatDefault returns null…

```suggestion
    const lines = splitLines(formatDefault(T[i]) ?? "").map(clipLine);
```